lib/helpers.js

```diff
@@ -15,6 +15,18 @@ if (Meteor.isClient){
 		}
 	});
 
+	Template.UserProfileSettings.helpers({
+		about: function () {
+			return Meteor.users.find( { _id: id } ).fetch()[0].profile.about;
+		},
+		email: function () {
+			return Meteor.users.find( { _id: id } ).fetch()[0].profile.email;
```

Comment:
'Meteor' is not defined.<br>'id' is not defined.<br>Too many errors. (7% scanned).